pkgs/by-name/gr/graphicsmagick/package.nix

```diff
@@ -0,0 +1,109 @@
+{ lib
+, bzip2
+, callPackage
+, coreutils
+, fetchurl
+, fixDarwinDylibNames
+, freetype
+, ghostscript
+, graphviz
+, libX11
+, libjpeg
+, libpng
+, libtiff
+, libtool
+, libwebp
+, libxml2
+, nukeReferences
+, quantumdepth ? 8
+, runCommand
+, stdenv
+, xz
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "graphicsmagick";
+  version = "1.3.43";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/graphicsmagick/GraphicsMagick-${finalAttrs.version}.tar.xz";
+    hash = "sha256-K4hYBzLNfkCdniLGEWI4vvSuBvzaEUUb8z0ln5y/OZ8=";
+  };
+
+  outputs = [ "out" "man" ];
+
+  patches = [
+    ./001-disable-popen.patch
```

Comment:
No patch comment?

Reply:
The name is self-explanatory.

Also, I don't know what is the status on commenting `patches` - very few people put it and many complain with the typical "where is it in the manual?", it quickly becomes tiresome...

Further, the code had no comments before, I merely changed its name to include the numbering.

Reply:
When reading a nix file, I put myself in the shoes of someone new, reading this nix file for the first time. And yes, I would like to have a comment explaining why this patch has been added and more importantly, when it is supposed to be removed (if applicable).

When doing review, I usually use this justification for missing patches: https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-patch-comment.md

As you can see, it hasn't been invented by myself, but by people that are much more knowledgeable than me when it comes to Nix.

Anyway, I don't want to spend time on this too much, if you deem that it's enough to be merged, just do it, I won't interfere any more in this PR.

Reply:
Thanks for adding a comment. I added the flag `security`, hope it's OK.